apps/transaction/forms/transaction_create_form.py

```
@@ -132,3 +135,8 @@ def _save_receipts(self, parent_transaction: ParentTransaction):
                 size=uploaded_file.size,
                 uploaded_by=getattr(self._request, "user", None),
             )
+
+    def _build_category_queryset(self):
```

Ruff (0.14.10)

Missing return type annotation for private function `_build_category_queryset`

(ANN202)